### ISSUE TITLE
Process stderr might be undefined

### ISSUE THREAD

**Issue by lppedd** [mike-lischke/antlr-tgen#1]
After spawning a process, you access `output.stderr`, but without checking if it exists.

https://github.com/mike-lischke/antlr-tgen/blob/95e0cd25790d95038c0da696a9275c7db75d0277/src/Generator.ts#L219

Without an additional check, e.g., `if (output.stderr && ...)`, I get

```
TypeError: Cannot read properties of null (reading 'length')
    at C.generateANTLRFilesInWorkDir
```

**Comment by mike-lischke** [comment]
There was no need to check. According to the typings this member cannot be null. I guess the type definitions need to be updated.

**Comment by lppedd** [comment]
Probably it also depends on the version of Node.js, but not sure. An additional check is never bad tho.

**Comment by mike-lischke** [comment]
Also fixed by 9f8c32a.
